src/api/config/routes.rb

```diff
@@ -210,6 +210,9 @@ def self.public_or_about_path?(request)
 
         resource :log, only: :show, controller: 'webui/cloud/upload_job/logs'
       end
+      resource :azure, only: [] do
```

Comment:
:thought_balloon: [scope](http://guides.rubyonrails.org/routing.html#prefixing-the-named-route-helpers)?

Reply:
@hennevogel 

> :thought_balloon: scope?

Done.

Reply:
@hennevogel 

> :thought_balloon: scope?

Moved to #4654.